src/meltano/core/plugin_install_service.py

```diff
@@ -379,9 +379,9 @@ async def install_plugin_async(
                 reason=reason,
                 status=PluginInstallStatus.ERROR,
                 message=(
-                    f"{plugin.type.descriptor} '{plugin.name}' "
+                    f"{plugin.type.descriptor.capitalize()} '{plugin.name}' "
                     f"could not be installed: {err}"
-                ).capitalize(),
+                ),
```

Comment:
This was capitalizing the full sentence, e.g.

```diff
- extractor 'tap-github' could not be installed: Some error
+ Extractor 'tap-github' could not be installed: some error
```

Notice how it changes the exception text to `some error`.